Description: fix sysv-install name to match the naming policy of tinysystemd
Author: Azer Abdullaev (Like All) <like-all@yandex.com>

--- a/src/systemctl/systemctl.c
+++ b/src/systemctl/systemctl.c
@@ -5153,7 +5153,7 @@
                 _cleanup_free_ char *p = NULL, *q = NULL, *l = NULL;
                 bool found_native = false, found_sysv;
                 unsigned c = 1;
-                const char *argv[6] = { ROOTLIBEXECDIR "/systemd-sysv-install", NULL, NULL, NULL, NULL };
+                const char *argv[6] = { ROOTLIBEXECDIR "/sysv-install", NULL, NULL, NULL, NULL };
                 char **k;
                 int j;
                 pid_t pid;
